src/test/java/org/folio/qm/controller/RecordsEditorRecordsApiTest.java

```diff
@@ -376,7 +375,7 @@ void testReturn401WhenHeader(){
     final var updateJobExecutionProfile = String.format(CHANGE_MANAGER_JOB_PROFILE_PATH, VALID_JOB_EXECUTION_ID);
     mockPut(updateJobExecutionProfile, SC_OK, wireMockServer);
 
-    final var error = verifyPost(recordsEditorPath(), quickMarcJson, SC_UNAUTHORIZED, JOHN_TOKEN_HEADER_INVALID).as(Error.class);
-    assertThat(error.getMessage(), equalTo("X-Okapi-Token does not contain a userId"));
+    final var error = verifyPost(recordsEditorPath(), quickMarcJson, SC_BAD_REQUEST, JOHN_TOKEN_HEADER_INVALID).as(Error.class);
+    assertThat(error.getMessage(), equalTo("User id can not be null"));
```

Comment:
Just to understand better, why do we prefer to validate for user id and not the x-okapi-token? I guess my point is if folks would be confused looking at the error message why a userId would be needed? I think if we say something like "Invalid x-okapi-token - userId cannot be null" - folks might understand better?

Reply:
I looked at the folio-spring-base changes and understand these better now. However, I still think the error message should be changed to reflect that there is a problem with the userId in the token - thanks!

Reply:
I agree that this message is better.
But on the other hand, there couldn't be a situation with an invalid token because okapi will not send a request with the invalid token.

Reply:
If I am understanding correctly, we are deducing the userId from x-okapi-token - right? so if a userId is null, then we could treat that as an invalid token?